src/dtm/DTM.ts

```diff
@@ -20,6 +19,16 @@ import {
 } from './DTM_transport';
 import { DtmModulationMode, DtmPacketType, DtmPhysicalLayer } from './types';
 
+const validate0x09Command = (res: number[] | undefined) => {
+    if (!Array.isArray(res) || res.length !== 2 || (res[1] & 0x01) !== 0) {
```

Comment:
does it mean that res[1] must be always non zero? maybe check can be simplified?

Reply:
Not sure about the simplified (like probably it can be, but I wanted to keep the check from the original validate function)
And yes, the least significant bit has to be 1. If it is 0 it indicates an error.

Reply:
`res[1] & 0x01` is more accurate as this make sure that if they has a values of 23 for example it is still success!